src/olympia/amo/tests/__init__.py

```diff
@@ -129,7 +129,7 @@ def check_links(expected, elements, selected=None, verify=True):
 
         e = elements.eq(idx)
         if text is not None:
-            eq_(e.text(), text)
+            assert e.text() == text
```

Comment:
Not actually essential for this commit but it gave me better debug information when a test was failing.  Left it in because everyone likes assert.